site/en/blog/deps-rems-106/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Deprecations and removals in Chrome 106
+description: >
+  A round up of the deprecations and removals in Chrome 106 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2022-09-01
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/EnYQIPXPy4yW6YavyMZu.png'
+alt: >
+  Deprecations and removals hero logo
+tags:
+  - deprecations-removals
+  - chrome-106
+---
+
+{% include 'partials/see-all-dep-rem.md' %}
+
+Chrome 106 beta was released on September 1, 2022 and is expected to become the stable version in late September, 2022.
+
+## Remove non-ASCII characters in cookie domain attributes {#remove-non-ascii-characters-in-cookie-domain-attributes}
```

Comment:
Anchors for headings need to be `{: #anchor }` and you missed the colon and space which was causing build to fail.

But honestly you don't need them - it'll auto generate them for you. I fixed this up recently where it wasn't stripping some punctuation characters out but should be all good now. So let's remove.

```suggestion
## Remove non-ASCII characters in cookie domain attributes
```

Reply:
I used a different markdown converter. That's what it did and it didn't occur to me to look at them closesly.